<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SupplementalListWidgetWithWidgets</class>
 <widget class="QWidget" name="SupplementalListWidgetWithWidgets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>325</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>256</width>
    <height>136</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>256</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* panel/background */

background: #404040;
box-shadow: inset 1px 0px 0px rgba(0, 0, 0, 0.25);

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>256</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>36</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgba(255, 255, 255, 0.0001);
box-shadow: inset 0px 1px 0px rgba(0, 0, 0, 0.2);</string>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>7</x>
          <y>9</y>
          <width>18</width>
          <height>18</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">/* icon/resource-image */


position: absolute;
width: 24px;
height: 24px;
left: calc(50% - 24px/2);
top: calc(50% - 24px/2);

color: rgba(255, 255, 255, 0.5);
background: rgba(255, 255, 255, 0.0001);</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/SVG/svg/icon/toc-resource.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Add">
        <property name="geometry">
         <rect>
          <x>225</x>
          <y>6</y>
          <width>24</width>
          <height>24</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Noto Sans</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">/* icon/reading-order-add-to-list */

/*
color: rgba(255, 255, 255, 0.5);
background: rgba(255, 255, 255, 0.0001);
*/

background: #404040;
border: 0px;
color: #FFFFFF;
</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/SVG/svg/icon/common-add.svg</normaloff>:/SVG/svg/icon/common-add.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>28</x>
          <y>6</y>
          <width>113</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">position: absolute;
width: 113px;
left: 28px;
top: 6px;
bottom: 6px;

font-family: Noto Sans TC;
font-style: normal;
font-weight: 600;
font-size: 12px;
line-height: 16px;
display: flex;
align-items: center;

/* panel/section title */
color: rgba(255, 255, 255, 0.5);
background: rgba(255, 255, 255, 0.0001);

</string>
        </property>
        <property name="text">
         <string>Supplemental List</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_NoItem">
       <property name="minimumSize">
        <size>
         <width>208</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>208</width>
         <height>15</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family: Noto Sans TC;
font-style: normal;
font-weight: normal;
font-size: 11px;
line-height: 15px;
/* identical to box height */

color: rgba(255, 255, 255, 0.45);
background: rgba(255, 255, 255, 0.0001);</string>
       </property>
       <property name="text">
        <string>     There is no supplemental item.</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QListWidget" name="listWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>256</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Noto Sans</family>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">
border:0px;

color: rgba(255, 255, 255, 0.5);
background: rgba(255, 255, 255, 0.0001);

margin: 0px;
spacing: 0px;

</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="autoScrollMargin">
        <number>0</number>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::InternalMove</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::MoveAction</enum>
       </property>
       <property name="resizeMode">
        <enum>QListView::Fixed</enum>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="uniformItemSizes">
        <bool>false</bool>
       </property>
       <property name="selectionRectVisible">
        <bool>true</bool>
       </property>
       <property name="itemAlignment">
        <set>Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
